boards/imix/src/main.rs

```diff
@@ -89,6 +90,11 @@ static mut RF233_BUF: [u8; radio::MAX_BUF_SIZE] = [0x00; radio::MAX_BUF_SIZE];
 static mut RF233_RX_BUF: [u8; radio::MAX_BUF_SIZE] = [0x00; radio::MAX_BUF_SIZE];
 static mut RF233_REG_WRITE: [u8; 2] = [0x00; 2];
 static mut RF233_REG_READ: [u8; 2] = [0x00; 2];
+
+// The MAC layer also requires a buffer to receive packets into. Some MAC
+// implementations like XMac may require an additional buffer to buffer packets.
+static mut MAC_RX_BUF: [u8; radio::MAX_BUF_SIZE] = [0x00; radio::MAX_BUF_SIZE];
```

Comment:
Why do we need a buffer _in addition_ to the radio rx buffer?

Reply:
You're right, that's not necessary. I've removed it.